ledger/daml-on-sql/README.rst

```diff
@@ -0,0 +1,742 @@
+lot of work which is both CPU- and IO-intensive: all (valid) Ledger API requests
+will eventually hit the database. At the same time, the Ledger API server has to
+have available resources to validate requests, evaluate commands and prepare responses.
+While the PostgreSQL schema is designed to be as efficient as possible, practical
+experience has shown that having **dedicated computation and memory resources for the
+two core components** (the Ledger API server and the PostgreSQL server) allows the two
+to run interfering as little as possible with each other. Depending on the kind of
+deployment you wish to make, this can be achieved with containerization, virtualization
+or simply using physically different machines. Still, the Ledger API communicates
+abundantly with the PostgreSQL server and many Ledger API requests need to go all
+the way to persist information on the database. To reduce the latency necessary to
+serve outstanding requests, **the Ledger API server and PostgreSQL server should be
+physically co-located**.
+
+Core availability considerations
+================================
+
+In order to address availability concerns, it's important to understand what each of the
+core components do and how they interact with each other, in particular regarding state
+and consistency.
+
+In DAML-on-SQL, the Ledger API server holds important state that affects the offset
+assigned to each transaction. This effectively means that having two Ledger API servers
+running on top of a single PostgreSQL server can lead to duplicated and inconsistent
+offsets, effectively leading to undefined (and likely broken) behavior. For this reason,
+it's important to maintain a strict 1:1 relationship between a running Ledger API server
+and a running PostgreSQL server. Note that using PostgreSQL in a high-availability
+configuration does not allow you to run additional Ledger API servers.
+
+Downtime for the Ledger API server can be minimized using a watchdog or orchestration
+system taking care of evaluating its health of the core components and ensuring its
+availability. The Ledger API implementation of DAML-on-SQL exposes the standard gRPC
+health checkpoint that can be used to evaluate the health status of the Ledger API
+component. More information on the endpoint can be found at the documentation for gRPC.
+
+Security and privacy
+********************
+
+Trust assumptions
+=================
+
+In DAML-on-SQL, all data is kept centrally by the operator of the deployment.
+Thus, it's their responsibility to ensure that the data is treated with the
+appropriate care so to respect confidentiality and the applicable regulations.
+
+The ledger operator is advised to use the tools available to them to not divulge
+private user data, including those documented by PostgreSQL, to protect data at
+rest and using a secure communication channel between the Ledger API server and
+the PostgreSQL server.
+
+Ledger API over TLS
```

Comment:
This section and a few below it seem to be relevant to any ledger API, not just DAML-on-SQL. Can we not just point to the relevant pages in the documentation (and improve them with the text below)?

Reply:
The JWT part is DAML-on-SQL specific, the rest is copy-pasted from the SDK docs. I'll link the definition of claims and preserve the reset.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6850/commits/2572ef91318bfa499ef48ab21865fe48fb48638d